notify/notify.go

```diff
@@ -251,6 +251,7 @@ type Metrics struct {
 	numTotalFailedNotifications        *prometheus.CounterVec
 	numNotificationRequestsTotal       *prometheus.CounterVec
 	numNotificationRequestsFailedTotal *prometheus.CounterVec
+	numAlertsSuppressedTotal           prometheus.Counter
```

Comment:
Apologies for not catching this earlier on, but yes, what we want here is `numNotificationSuppressedTotal` it's a bit confusing but we have strict nomenclature at a metric level and `notification` is not the same thing as an `alert`.

Alerts are what the Alertmanager receives and as you correctly figured out we already reflect those with the suppressed label on `alertmanager_alerts`.

What we want here is `alertmanager_notifications_supressed_total`

Reply:
There's also an argument on wether we should use a label to specify if this was suppressed by a an `active_interval` or a `muted_interval` but let's leave this out for now and see wether this is useful on its own as is.

Reply:
Ok, I like that distinction much better, I'll move towards this phrasing. 

> There's also an argument on wether we should use a label to specify if this was suppressed by a an active_interval or a muted_interval but let's leave this out for now and see wether this is useful on its own as is.

Agreed, and sounds good to me :+1:

Reply:
addressed in e15292f